x/concentrated-liquidity/types/msgs_test.go

```diff
@@ -31,6 +31,7 @@ func init() {
 }
 
 func runValidateBasicTest(t *testing.T, name string, msg extMsg, expectPass bool, expType string) {
+	t.Helper()
```

Comment:
automatically letting us know when we can get more info when runnning tests by marking a function as a helper function